lib/index.js

```diff
@@ -17,6 +17,15 @@ const realCreateWriteStream = fs.createWriteStream;
 const realStats = realBinding.Stats;
 const realStatWatcher = realBinding.StatWatcher;
 
+// File Permissions
+const S_IRUSR = 0o400;
+const S_IRGRP = 0o40;
+const S_IROTH = 0o4;
+
+const S_IXUSR = 0o100;
+const S_IXGRP = 0o10;
+const S_IXOTH = 0o1;
```

Comment:
`lib/item.js` has these permissions defined but not exported, you can export them.